frontend/src/interfaces/types.ts

```diff
@@ -34,3 +34,11 @@ export interface UserType {
   jwtToken: string
   role: number
 }
+export interface RegisterFormData {
+  terms: boolean;
```

Comment:
Booleans should be named as questions: `areTermsRead`, `areTermsConfirmed`, `areTermsChecked`